platform/ios/src/MGLMapView.mm

```diff
@@ -1128,9 +1128,68 @@ - (void)wakeGL:(__unused NSNotification *)notification
         [self validateLocationServices];
 
         [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
+        
+        [self updateLocalization];
+    }
+}
+
+- (void)updateLocalization
+{
+    NSString *mapboxLanguage = [[NSUserDefaults standardUserDefaults] stringForKey:@"MGLMapboxLanguage"];
```

Comment:
I’m not sure `NSUserDefaults` is appropriate here — it’s possible that a user will load a map, we will localize it once, then the app will be eventually be killed. When the app restarts, we would not relocalize the map because the key was never reset.

Instead, I think an ivar might work better.